test/unit/controllers/pages/student.js

```diff
@@ -127,6 +127,53 @@ describe("StudentController", () => {
         });
     });
 
+    describe("checkInModel", () => {
```

Comment:
Make sure to add test for the checkInModel returning a rejected promise after adding the error catch to the controller.